server/http_status.go

```diff
@@ -44,6 +44,9 @@ func (s *Server) startHTTPServer() {
 	// HTTP path for prometheus.
 	router.Handle("/metrics", prometheus.Handler())
 
+	// HTTP path for dump statistics.
+	router.Handle("/dump_stats/{db}/{table}", s.newStatsHandler())
```

Comment:
How about use `/stats/{cmd}/{db}/{table}`?  For now, `dump` is the only command. We could extend it in the future.